File: conexiones/conexion.rb/mongo.rb
## instalar desde terminal gem install mongo

require 'mongo'

begin
  cliente = Mongo::Client.new('mongodb://localhost:27017/')

  puts cliente
  puts "Conexión exitosa"

  puts "Tipo de conexión: #{cliente.class}"

  puts "Bases de datos disponibles: #{cliente.database_names}"

rescue Mongo::Error => e
  puts "Error al conectar a MongoDB: #{e.message}"
end

File: conexiones/conexion.js/mongo.js
//el primer paso es instalar npm install mongodb//
// Importamos el paquete mongodb
const { MongoClient } = require('mongodb');

// URI de conexión a MongoDB
const uri = "mongodb://localhost:27017/";

// Conexión a MongoDB
async function conectarMongoDB() {
    try {
        // Crear cliente y conectar a MongoDB
        const cliente = new MongoClient(uri);
        await cliente.connect();

        console.log(cliente);
        console.log("Conexión exitosa");

        console.log(`Tipo de conexión: ${cliente.constructor.name}`);

        const basesDeDatos = await cliente.db().admin().listDatabases();
        console.log("Bases de datos disponibles:", basesDeDatos.databases);

        // Cerrar la conexión
        await cliente.close();
    } catch (error) {
        console.error("Error al conectar a MongoDB:", error);
    }
}

conectarMongoDB();
